test/model_factory_test.py

```diff
@@ -73,6 +85,12 @@ def setUp(self):
           model:
             cfg_type_name: a_type
             type: NamedModelElement
+        unused_type:
+          src:
+          - file: unused_type_values.xxx
+          model:
+            cfg_type_name: unused_type
+            type: NamedModelElement
```

Comment:
same here: is the type actually required? Could still make sense to leave it int, even if it is not required